contracts/strategy/trade-fintech/TradeFintechStrategy.sol

```diff
@@ -109,6 +111,7 @@ contract TradeFintechStrategy is BaseStrategy, ITradeFintechStrategy {
 
     /// @inheritdoc BaseStrategy
     function _freeFunds(uint256 amount) internal override {
+        if (block.timestamp > lockPeriodEnds) revert LockPeriodEnded();
         uint256 invested = totalInvested;
```

Comment:
`totalInvested` is not 0 because during the deposit strategy deploys funds to the manager, but in general it's correct, I will restrict it to deposit period. `_freeFunds` suppose to work only during the deposit period, after lock period end manager calls the `repay` fn all the fund returns to the strategy with profit or loss